inscriptions/ISic001265.xml

```diff
@@ -216,13 +216,12 @@
             </div>
             <div type="bibliography">
                 <listBibl type="edition">
-                    <bibl type="corpus" n="IG">
-                        <citedRange>
-                            <ref target="https://clio.columbia.edu/catalog/6873760">14.0440</ref>
-                        </citedRange>
-                        <ptr target="https://www.zotero.org/groups/382445/items/Q2SBPG9F"/>
```

Comment:
great to add the Ferrua reference, but I don't understand why you have deleted the IG reference? We aim to include a comprehensive bibliography, and we will list all known editions wherever possible, not simply the one that we are using (and that is why, in turn, we specify which one we are currently reporting in the apparatus).